doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -26,9 +26,9 @@
 <a name="3.1"></a>
 ## 3.1 Introduction
 
-The Common Telco NFVI OpenStack Reference Architecture (RA) aims to provide an industry standard reference architecture independent of the many distributions of OpenStack. It does not seek to change any vendor implementation assuming Common Telco NFVI compliance out of the box without vendor specific enhancements that are not up-streamed. This would allow operators to provide a common OpenStack-based architecture allowing any compliant VNF to be deployed and operate as expected.  The purpose of this chapter is to outline all the components required to provide the NFVI and the VIM in a consistent and reliable way. 
+This CNTT Reference Architecture (RA-1) aims to provide an OpenStack distribution agnostic reference architecture. The different OpenStack distributions, without the not up-streamed vendor specific enhancements, are assumed to be CNTT conformant. This Reference Architecture allows operators to provide a common OpenStack-based architecture for any CNTT compliant VNF to be deployed and operated as expected.  The purpose of this chapter is to outline all the components required to provide the NFVI and the VIM in a consistent and reliable way. 
```

Comment:
CNTT is a transitory term...

Reply:
@iangardner22 I propose to not mix these minor PRs with another discussion.

Reply:
Do we have already a discussion about using the term CNTT in the reference documents somewhere? Should be clarified in general but most probably not in context of this PR.

Reply:
@iangardner22 @hdamker Need a TSC decision and if CNTT is not be used then all documents would need to be edited.

Reply:
@hdamker @pgoyal01 totally agree!  Reason I am picking it up in these PR is because it was pointed out to me previously.   @rabi-abdel is aware and putting on next meeting agenda